index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+     
+         <div class="header">
+               <div class="logo">
+                   <h1>My Logo</h1>
+               </div>
```

Comment:
- [ ] Please I highly recommend wrapping a link `a` tag to your "My Logo" and remove the `h1` tag for better implementation of HTML semantics and user experience. Please [check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more information. 😄